src/__tests__/multiStore.test.js

```diff
@@ -30,6 +50,10 @@ describe('multiStore', () => {
       sessionStorage.setItem('foo', 'baz')
       expect(stores.get('foo')).toBe('baz')
     })
+    test('calls the error handler if an error is thrown', () => {
+      errorStores.get('foo')
+      expect(errorHandler).toBeCalledTimes(1)
```

Comment:
It would also be interesting to ensure you have access to the passed data ([`toHaveBeenCalledWith`](https://jestjs.io/docs/en/expect#tohavebeencalledwitharg1-arg2-)). Same for the others.